src/nv_ingest/framework/orchestration/morpheus/util/pipeline/pipeline_runners.py

```diff
@@ -62,9 +62,6 @@ class PipelineCreationSchema(BaseModel):
     message_client_port: str = "7671"
```

Comment:
One overall comment on anything under `framework/orchestration/morpheus`

These shouldn't be updated to anything else, they're meant to be Morpheus specific. If we want to remove all of the associated code, we can prune the full orchestration branch.

Reply:
I'm going to remove it all for now. Since morpheus dependency is now removed leaving it doesn't really buy us anything.